connectors/cache/fallback.go

```diff
@@ -60,6 +60,22 @@ func WithSkipWriteInvalidateEntities(entities ...dosa.DomainObject) Options {
 	}
 }
 
+// Context key used by SetCacheableEndpoints
+var (
+	// ContextEndpoint allows users to pass in calling endpoint name
+	ContextEndpoint string = "endpoint"
+)
+
+// SetCacheableEndpoints sets cacheable endpoints
+func SetCacheableEndpoints(endpoints ...string) Options {
+	return func(c *Connector) error {
+		for _, endpoint := range endpoints {
+			c.cacheableEndpoints[endpoint] = true
```

Comment:
I would expect an array named `cacheableEndpoints` to contain endpoints; not the status of endpoints. This should be called something like `cacheableEndpointStatus` (although that is a very long name).

I'm also not a fan of bool vars, an enum with descriptive names is more readable. I won't insist on that, though -- I'll leave it between you and your conscience.

Reply:
Also, please add a line to the CHANGELOG describing the change.

Reply:
Done!